### ISSUE TITLE
from_raster conflicts with wildfire init

### ISSUE THREAD

**Issue by carmensteinmann** [CLIMADA-project/climada_petals#25]
Hello! 
I came across a similar problem as discussed in issue #15 when setting up a method to read raster files into the wildfire hazard (I'm working on the feature branch feature/wildfire).

This was the error: 

```
  File "/Users/carmensteinmann/Documents/CLIMADA/climada_petals/climada_petals/hazard/wildfire.py", line 1311, in from_netcdf
    haz = cls.from_raster([str(Path(input_dir, filename))], band=id_bands,

  File "/Users/carmensteinmann/Documents/CLIMADA/climada_python/climada/hazard/base.py", line 289, in from_raster
    haz = cls(pool=pool)

TypeError: __init__() got an unexpected keyword argument 'pool'
```

Which could be fixed by adapting the initiation of the wildfire hazard to:
    
```
def __init__(self, pool=None):
        """Empty constructor."""
        Hazard.__init__(self, HAZ_TYPE)
        if pool:
            self.pool = pool
            LOGGER.info('Using %s CPUs.', self.pool.ncpus)
        else:
            self.pool = None
            
        self.FirmsParams = self.FirmsParams()
        self.ProbaParams = self.ProbaParams() 
```

Is this the way to go?

**Comment by tovogt** [comment]
A typical way how to solve this would be:

```
def __init__(self, **kwargs):
        """Empty constructor."""
        Hazard.__init__(self, HAZ_TYPE, **kwargs)
        self.FirmsParams = self.FirmsParams()
        self.ProbaParams = self.ProbaParams() 
```

**Comment by carmensteinmann** [comment]
Thanks @tovogt for the quick answer! 
I implemented your suggestion in #29.
